front_end/entrypoints/rn_fusebox/rn_fusebox.ts

```diff
@@ -74,6 +74,15 @@ RNExperiments.RNExperimentsImpl.Instance.enableExperimentsByDefault([
   Root.Runtime.ExperimentName.REACT_NATIVE_SPECIFIC_UI,
 ]);
 
+document.addEventListener('visibilitychange', () => {
+  const visibilityState = document.visibilityState;
+  if (visibilityState !== 'visible' && visibilityState !== 'hidden') {
```

Comment:
`visibilityState.prerender` has been deprecated, so this should be redundant now - https://developer.mozilla.org/en-US/docs/Web/API/Document/visibilityState

Reply:
Nice catch! The Flow type still has `prerender` and `unloaded` but should be irrelevant for Chrome here